test/unit/org/apache/cassandra/repair/RepairJobTest.java

```diff
@@ -91,14 +95,16 @@
     private static InetAddressAndPort addr3;
     private static InetAddressAndPort addr4;
     private static InetAddressAndPort addr5;
-    private RepairSession session;
+    private MeasureableRepairSession session;
     private RepairJob job;
     private RepairJobDesc sessionJobDesc;
 
     // So that threads actually get recycled and we can have accurate memory accounting while testing
     // memory retention from CASSANDRA-14096
     private static class MeasureableRepairSession extends RepairSession
     {
+        List<Callable<?>> onSyncComplete = new ArrayList<>();
```

Comment:
nit: Make it `final`
naming nit: perhaps something like `syncCompleteCallbacks` or `completionCallbacks`

Reply:
+1 changed to `syncCompleteCallbacks`